package.json

```diff
@@ -62,12 +59,10 @@
     "@types/react-dom": "^16.9.10",
     "babel-loader": "^8.2.1",
     "classnames": "^2.2.6",
-    "conventional-changelog-conventionalcommits": "^4.4.0",
-    "css-loader": "^5.0.1",
```

Comment:
Sure this won't break loading `.css` files?

Reply:
Everything builds and storybook runs so for now yes. I never had to use a css loader in the past for loading CSS, I'm guessing it was always handled by the bundler, so maybe it could be done as part of the bundler config issue for example.